modules/react/tabs/lib/menu/MenuList.tsx

```diff
@@ -39,9 +39,10 @@ export const MenuList = createComponent('div')({
         borderRadius="m"
         padding="zero"
         marginY="zero"
-        marginX="xxs"
+        marginX="zero"
```

Comment:
This could probably just be`margin="zero"` instead of using `marginX` and `marginY`.